etc/calico-gen-bird6-conf.sh

```diff
@@ -18,7 +18,7 @@ where
   <my-ipv6-address> is the external IPv6 address of the local machine
   <rr-ipv6-address> is the IPv6 address of the route reflector that
       the local BIRD6 should peer with
-  <as-number> is the BGP AS number that the route relector is using.
+  <as-number> is the BGP AS number that the route reflector is using.
```

Comment:
I agree in principle, but I don't think I can do and test the corresponding IPv6 changes in the same timescale in which I'd like to release the IPv4 mesh support (i.e. today, really).

Surely it's also useful to release IPv4 mesh support, and then do a delta for IPv6?